src/Sidekick/Windows/LeagueOverlay/LeagueOverlayController.cs

```diff
@@ -5,42 +5,56 @@
 using System.Threading.Tasks;
 using System.Windows.Forms;
 using System.Windows.Input;
+using Sidekick.Core.Natives;
+using Sidekick.Localization;
 using Cursor = System.Windows.Forms.Cursor;
 
 namespace Sidekick.Windows.LeagueOverlay
 {
-    public static class LeagueOverlayController
+    public class LeagueOverlayController: IDisposable
```

Comment:
We want to move away from using Controllers. We want to use ViewModels that are located in the UI project. You can take a look at the SettingsViewModel and View to see how to make it work.

Reply:
Domialex is okay with merging as it is. But more work will need to be done to move most of the code to view models. Let me know if you want to fix this now or if you want us to merge.